bedrock/mozorg/templates/mozorg/about/includes/m24/idea.html

```diff
@@ -4,14 +4,13 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
-<div class="m24-c-transition m24-t-purple"><hr></div>
-<div class="m24-t-purple">
+<div class="m24-t-green">
   <section class="m24-c-showcase">
-    <h2 class="m24-c-showcase-title">IDEA ipsum dolor sit amet</h2>
-    <p class="m24-c-showcase-subtitle">Suspendisse commodo sed erat in pellentesque. In sit amet tortor porttitor, gravida mauris tempor, pharetra erat. Nam vel justo urna.</p>
+    <h2 class="m24-c-showcase-title">Building a fairer future</h2>
+    <p class="m24-c-showcase-subtitle">See how we’re fostering a diverse, inclusive and accessible online environment for all.</p>
     <div class="m24-c-showcase-media">
       <img src="{{ static('img/m24/todo-1-1.svg') }}" alt="" height="500" width="500">
     </div>
-    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.impact-report.index') }}" class="m24-c-cta" data-cta-text="IDEA">Read the report</a></p>
+    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.impact-report.index') }}" class="m24-c-cta" data-cta-text="IDEA">See more</a></p>
```

Comment:
**Question:** Do we also need to update `data-cta-text` here, or is `IDEA` still a relevant value?

Reply:
Question: Also, is `idea.html` still the most suitable filename?

Reply:
I have update to use `impact` here, but I am open to better options